### PR TITLE
use mos to generate and test the readme

### PR DESCRIPTION
**Pull request by zkochan**
After I've added mos to generate and test the readme, I've found out that the example in the readme wasn't working.

It won't happen again. Now the readme example is tested by mos

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -62,28 +78,62 @@ You can check the [default reporter](lib/reporter.js) to get the gist of how it
...
-readPkgUp
+readPkgUp()
   .then(function(result) {
      var scriptsInfo = getScriptsInfo(result.pkg);
-     console.log(result.scriptsInfo);
-     //> { test: 'Run the tests' }
+     console.log(scriptsInfo);
+     //> { md: 'Update the markdown files',
+     //    info: 'Display information about the scripts',
+     //    test: 'Run the tests' }
```
Comment (srph) [inline readme.md:93]: I think this is better off redacted.

``` js
console.log(scriptsInfo); // { md: 'Update the markdown files', info: ..., test: ... }
```

Comment (zkochan) [inline readme.md:93]: Currently I cannot do it. That comment is inserted automatically by [this library](https://github.com/zkochan/codemo) that combines code with output. It currently inserts the output only on next line. All I can do is this version, if you like it more:

``` js
console.log(JSON.stringify(scriptsInfo));
//> {"md":"Update the markdown files","info":"Display information about the scripts","test":"Run the tests"}
```

**Comment by srph** [comment]
Sorry for the delays, been busy. Just to make sure, since this is just a readme (unless I missed anything), we don't have to make any releases, right? :)

**Comment by zkochan** [comment]
No problem:)

Right, you can release only if you want to update the readme on npm

**Pull request merged**